aiobotocore/client.py

```diff
@@ -6,16 +6,50 @@
 import botocore.client
 import botocore.serialize
 import botocore.validate
-from botocore.exceptions import ClientError
-from botocore.exceptions import OperationNotPageableError
+import botocore.parsers
+from botocore.exceptions import ClientError, OperationNotPageableError, ParamValidationError
 from botocore.paginate import Paginator
 from botocore.signers import RequestSigner
 
 from .paginate import AioPageIterator
-
 from .endpoint import AioEndpointCreator
 
 
+class AioConfig(botocore.client.Config):
+    def __init__(self, connector_args=None, **kwargs):
+        super().__init__(**kwargs)
+
+        self._validate_connector_args(connector_args)
+        self.connector_args = copy.copy(connector_args)
+
+        if 'keepalive_timeout' not in connector_args:
+            # AWS has a 20 second idle timeout: https://forums.aws.amazon.com/message.jspa?messageID=215367
+            # and aiohttp default timeout is 30s so we set it to something reasonable here
+            self.connector_args['keepalive_timeout'] = 12
+
+    @staticmethod
+    def _validate_connector_args(connector_args):
+        if connector_args is None: return
+
+        # We could do this with voluptuous as well but would require another module
+        for k, v in connector_args.items():
+            if k in ['use_dns_cache', 'verify_ssl']:
+                if not isinstance(v, bool):
+                    raise ParamValidationError(report='{} value must be a boolean'.format(k))
+            elif k in ['conn_timeout', 'keepalive_timeout']:
+                if not isinstance(v, float) and not isinstance(v, int):
+                    raise ParamValidationError(report='{} value must be a float/int'.format(k))
+            elif k == 'limit':
+                if not isinstance(v, int):
+                    raise ParamValidationError(report='{} value must be an int'.format(k))
+            elif k == 'ssl_context':
+                import ssl
```

Comment:
I figured you don't want to import modules you don't need due to memory implications, and since this is a built-in module it shouldn't break scripts.  I can move it to the top if you'd like though.